src/store/index.js

```diff
@@ -1,14 +1,33 @@
 import { createStore } from 'redux';
 import reducer from '../reducers';
-import { saveState } from '../utils';
+import { resetData } from '../actions';
+import getInitialState from './initial-state';
+import loadData from './load-data';
+import { saveState } from './helpers';
+import checkFontLoaded from './check-font-loaded';
+
+export default function configureStore(props) {
```

Comment:
Can you outline the methods functionality in a JSDoc (particularly outlining the expected data to be passed to the method)? Also is there any expected names in the object being passed e.g. "data" is one - I ask because maybe this method could be updated like:

```
export default function configureStore({
  data,
  someOtherProp,
  anotherAwesomeProp,
  etcProp
}) {
```

Reply:
JSDoc is a good shout, thanks, will do.

Easiest to keep props intact as I'm just passing the whole thing to getInitialState and I think it's less brittle not to deconstruct then immediately reconstruct it?

Reply:
Cool, JSDoc description of the expected data structure being passed down would be great 🙂